src/core/perf_stats.cpp

```diff
@@ -38,12 +40,17 @@ PerfStats::~PerfStats() {
     std::ostringstream stream;
     std::copy(perf_history.begin() + IgnoreFrames, perf_history.begin() + current_index,
               std::ostream_iterator<double>(stream, "\n"));
-    const std::string& path = Common::FS::GetUserPath(Common::FS::UserPath::LogDir);
+
+    const auto path = Common::FS::GetYuzuPath(Common::FS::YuzuPath::LogDir);
     // %F Date format expanded is "%Y-%m-%d"
-    const std::string filename =
-        fmt::format("{}/{:%F-%H-%M}_{:016X}.csv", path, *std::localtime(&t), title_id);
-    Common::FS::IOFile file(filename, "w");
-    file.WriteString(stream.str());
+    const auto filename =
+        path / fmt::format("{:%F-%H-%M}_{:016X}.csv", *std::localtime(&t), title_id);
```

Comment:
makes it look more tidy, and more accurate variable names.
```suggestion
    const auto filename = fmt::format("{:%F-%H-%M}_{:016X}.csv", *std::localtime(&t), title_id);
    const auto filepath = path / filename;
```